.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Release
+on:
+  push:
+    tags:
+      - "v**"
+jobs:
+  build:
+    defaults:
+      run:
+        working-directory: src/Meplato.Store2
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    # ----- SETUP
+    - name: Setup MSBuild path
+      uses: microsoft/setup-msbuild@v1.0.0
+    - name: Setup NuGet
+      uses: NuGet/setup-nuget@v1.0.2
+    # ----- BUILD
+    # Do not use `dotnet restore`, it ignores packages.config
+    - name: Restore NuGet packages
+      run: nuget restore -SolutionDir ..
+    - name: Build
+      run: msbuild -t:Build -p:Configuration=Release
+    # ----- NUGET RELEASE
+    - name: Build Nupgks
+      run: |
+        $tag = ${env:GITHUB_REF}.Substring(11)
+        nuget pack Meplato.Store2.csproj -Properties Configuration=Release -Version ${tag}
```

Comment:
GITHUB_REF = `refs/heads/<tag_name>` of which we remove chars from 0-11, resulting in `tag_name`